src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/javaparser/InputJavaParserNoFreezeOnDeeplyNestedLambdas.java

```diff
@@ -0,0 +1,29 @@
+//non-compiled with javac: exceeds time limits in CI
```

Comment:
This bug was fixed in JDK10:
https://bugs.java.com/bugdatabase/view_bug.do?bug_id=8078093

Please try `Compilable with Java16` here. It compiles fine on my PC with JDK16.

Reply:
Awesome, this worked. Thanks a lot!